deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/multilayer/MultiLayerNetwork.java

```diff
@@ -2751,6 +2762,26 @@ public Evaluation evaluate(DataSetIterator iterator, List<String> labelsList) {
         return evaluate(iterator, labelsList, 1);
     }
 
+    @Override
+    public INDArray updaterState() {
+        return getUpdater() != null ? getUpdater().getStateViewArray() : null;
+    }
+
+    @Override
+    public void fit(MultiDataSet dataSet) {
+        throw new DL4JInvalidInputException("MultiLayerNetwork can't handle MultiDataSet. Please consider use of ComputationGraph");
```

Comment:
Not a big deal, but we *could* if we want check for single feature array + single label array, and convert to DataSet. And wrap the MultiDataSetIteartor in a DataSetIterator
(I don't mind doing this post merge of this; we can open an issue for that unless you want to tackle it now)